portal/app/settings/base.py

```diff
@@ -121,3 +121,6 @@
 }
 SUN_TECH_DRIVE_URL = ''
 SUN_TECH_DRIVE_TEST_URL = 'https://test.suntechdrive.com'
+
+EGAUGE_URL='https://egauge46613.egaug.es/cgi-bin/egauge?param'
```

Comment:
this dangling `param` is weird. Is it needed? Do we append anything to this URL?

Reply:
this is part of the URI. Just tested removing it and it seems to have the same behavior with or without it. So technically we could remove it I guess.

Reply:
Yeah, query string params are generally in the format `?param_name=param_value`. The above doesn't have a value, so it shouldn't do anything.

Reply:
ok, didn't know that. I'll remove then